src/ui/ResultLayout/ResultLayout.ts

```diff
@@ -37,6 +39,7 @@ export class ResultLayout extends Component {
    * The possible valid and supported layout.
```

Comment:
I didn't remove it, but exposing this in the doc doesn't actually show the supported values in the doc site. Would you like me to un-expose this?